### DISCUSSION TITLE
Copilot not working in VS Code

### DISCUSSION THREAD

**Discussion by ExcitedDeveloper** [community/community#6810] [Copilot Conversations]
Copilot is not working for me.  I don't get any suggestions.  I did receive an invite email.  I installed the extension.  I see the Copilot icon in the bottom right.  

When I go to View -> Output -> GitHub Copilot, I see errors like the following:

    Request Error: unable to get local issuer certificate

Not sure if this has anything to do with it, but I am on my work computer going through a VPN.

Has anybody seen this error and been able to resolve it?

**Comment by davecheney** [comment]
@ExcitedDeveloper i think something on your computer, or your network, is intercepting your SSL traffic.

**Comment by myn** [comment]
Same issue here.  Large enterprise with McAfee Client Proxy (MCP) running locally.  I've disabled "Http: Proxy Strict SSL" in vs.code settings however it appears this extension is not abiding by this.

**Comment by ghost** [comment]
Same issue here.... Below is the output text:


[INFO] [default] [2021-10-27T22:34:55.355Z] [fetchCompletions] engine https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex
[ERROR] [default] [2021-10-27T22:34:55.769Z] Request Error: unable to get local issuer certificate
[ERROR] [ghostText] [2021-10-27T22:34:55.771Z] Error on ghost text request Error: unable to get local issuer certificate
[INFO] [ghostText] [2021-10-27T22:34:55.771Z] Breaking, no choices



Not sure what to look for next

**Reply by davecheney** [reply]
Do you have some kind of VPN or security software on your machine which is intercepting SSL connections?
